README.md

```diff
@@ -33,28 +40,56 @@ export GCLOUD_SERVICEACCOUNT_CLIENT_EMAIL=<client-email present in service-accou
 export GRAFANA_ADMIN_PASSWORD=password
 export DOMAIN_NAME=prombench.prometheus.io
```

Comment:
`export DOMAIN_NAME=prombench.prometheus.io // Can be set to any other custom domain.`